get-tom-observations/main.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sqlalchemy
+from flask import jsonify
+
+# Set the following variables depending on your specific
+# connection name and root password from the earlier steps:
+connection_name = os.getenv('CONNECTION_NAME')
+db_password = os.getenv('DB_PASSWORD')
+db_name = os.getenv('DB_NAME')
+
+db_user = os.getenv('DB_USER')
+driver_name = 'postgres+pg8000'
```

Comment:
Is this required by TOM?  The google documentation doesn't seem to require this driver so just wondering if necessary.

https://cloud.google.com/sql/docs/mysql/connect-functions#connecting_to

Reply:
You're right. I think `psycopg2` is used by default, and TOM doesn't ask for a specific driver name.